vis.go

```diff
@@ -2,13 +2,12 @@ package mtree
 
 // #include "vis.h"
 import "C"
-import "fmt"
 
-func Vis(str string) (string, error) {
-	dst := new(C.char)
-	ret := C.strvis(dst, C.CString(str), C.VIS_WHITE|C.VIS_OCTAL|C.VIS_GLOB)
-	if ret == 0 {
-		return "", fmt.Errorf("failed to encode string")
-	}
-	return C.GoString(dst), nil
+// Vis is a wrapper of the C implementation of the function vis, which encodes
+// a character with a particular format/style
+func Vis(str string) string {
+	// dst needs to be 4 times the length of str (plus 1 for null char)
+	dst := C.CString(string(make([]byte, 4*len(str)+1)))
```

Comment:
I tested this on your branch and it worked. And there is no catching the SIGSEGV